active/website.md

```diff
@@ -0,0 +1,62 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Members can work on bugs
+- Members triage the project’s issues and pull requests.
+- Member maintain and monitor the website including updating versions
+- Mentor new contributors to the website
+- Chair, Co-Chair and Board Liaison can sign off on new features
+
+## Initial membership
+
+- Chair: TBA
+- Co-Chair: TBA
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBA
+- Other members:
+    - Sarah Abderemane
+    - Eric Sherman
+    - Mark Walker
+    - Jason Judkins
+    - Paolo Melchiorre
+    - Sanyam Khurana
+    - Tobias McNulty
+    - Ron Maravanyika
```

Comment:
This is missing a few members from the current maintainers group.

Reply:
@cgl yes that is correct. This list is just a list of individuals who confirmed that they wanted to be part of the working group. Because there are some individuals who are part of the current maintainers group that don't want to be part of the working group as they won't be contributing going forward.